src/stuf/stuf_bg/checks.py

```diff
@@ -0,0 +1,59 @@
+from django.utils.translation import gettext_lazy as _
+
+import requests
+from lxml import etree
+
+from openforms.plugins.exceptions import InvalidPluginConfiguration
+from openforms.utils.xml import fromstring
```

Comment:
the `stuf` package cannot know about the `openforms` package

`src/stuf` is a candidate to be published on PyPI when other projects also do things with stuf, here especially you have to respect the allowed public APIs